windows/src/desktop/kmshell/Makefile

```diff
@@ -5,8 +5,7 @@
 !include ..\..\Defines.mak
 
 build: version.res manifest.res icons xml
-    $(MKVER_U) localedef.dtd.in xml\localedef.dtd $(MKVER_VERSION_TXT)
-    $(PROGRAM)\buildtools\mkver -v $(MKVER_VERSION_TXT)
```

Comment:
I assume it's safe to drop the second line here?  I'm kinda skimming due to the PR's size, and some similar areas elsewhere seemed to keep this.

Reply:
Yeah, this was handled with the `version.res` target, so shouldn't have been there in the first place.